about.html

```diff
@@ -0,0 +1,216 @@
+        <ul
+          id="nav-mobile"
+          class="flex flex-col hidden md:hidden justify-between fixed gap-6 h-[85vh] text-5xl text-white py-24 font-bold z-20"
+        >
+          <li><a href="./about.html">About</a></li>
+          <li><a href="index.html#program">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="index.html#sponsors">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- Hero -->
+    <section
+      id="hero"
+      class="bg-bgHero md:bg-bgHeroDesktop bg-no-repeat bg-cover min-h-[600px]"
+    >
+      <div class="container mx-auto px-[2%] py-24">
+        <div class="flex flex-col items-center space-y-12">
+          <div class="flex flex-col items-center gap-2 text-center">
+            <span class="text-callAction text-xl font-medium md:text-3xl"
+              >"Hello! Brillian Minds"
+            </span>
+            <h1
+              class="text-4xl md:text-6xl md:w-[90%] lg:w-[70%] text-callAction font-[900] bg-textImg"
+            >
+              CREATIVE COMMONS GLOBAL SUMMIT 2022
+            </h1>
+          </div>
+          <div
+            class="px-4 py-6 border-2 border-gray-300 bg-white md:w-[90%] lg:w-[74%]"
+          >
+            <p class="text-gray-700 text-center font-semibold">
+              Motivational speeches can provide inspiration, boost morale, and
+              help individuals overcome obstacles. They offer practical tools
+              and strategies to help people achieve their goals and dreams. They
+              can also help individuals build self-confidence, improve
+              communication skills, and foster a positive mindset.
+            </p>
+          </div>
+          <div class="text-center">
+            <p class="mt-2">
+              Please contact us by Email for your ticket reservation, and more
+              questions about Ex-Investor Event
+            </p>
+            <span class="text-primary font-bold underline underline-offset-4">
+              contact@ex-investor.com
+            </span>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Logo Section -->
+    <section class="border-b">
+      <div class="container flex flex-col mb-16 mx-auto space-y-8">
+        <div class="flex flex-col gap-4 items-center pt-8 px-2">
+          <h2 class="text-primary text-2xl font-medium">
+            Ex-investor - Boost The Mind Logo
+          </h2>
+          <span class="dash bg-callAction"></span>
+          <p class="text-center">
+            the logo of Ex-Investor was made by @Yusuf.obr on 15,Mars 2023 this
+            make with mixing beautiful colors
+          </p>
+        </div>
+        <div class="p-8 border w-[80%] mx-auto">
+          <img
+            class="max-w-[200px] mx-auto"
+            src="./images/LOGO.svg"
+            alt="ex-investor"
+          />
+        </div>
+      </div>
+    </section>
```

Comment:
- Kindly, adjust those titles to the center to stick to the given design and have a better design 👌
👇 your design
![2023-03-16_15-10](https://user-images.githubusercontent.com/52010789/225615576-7c4945c7-312f-46ef-b4fb-6b6e62cb27c8.png)
👇 given design
![2023-03-16_15-21](https://user-images.githubusercontent.com/52010789/225615839-f2339a4d-5aad-4ef2-989d-2050a0e6d7b2.png)

Reply:
done